npc/custom/woe_controller.txt

```diff
@@ -190,7 +194,7 @@ OnReward:
 				}
 			}
 		}
-	if (.Options&2) query_sql("UPDATE `mail` SET message = REPLACE(message,'%',CHAR(13)) WHERE send_name = 'no-reply'");
```

Comment:
I think that if you delete this, then the .@body$ variable should include new lines instead of % symbols

Reply:
You're right !